cmd/relay.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/maintainer"
+	"github.com/spf13/cobra"
+)
+
+var maintainerConfig = &maintainer.Config{}
+
+// RelayCommand contains the definition of the relay command-line subcommand.
+var RelayCommand = &cobra.Command{
```

Comment:
I think we must think about this command a little bit wider. The package `pkg/maintainer` will hold not only the relay maintainer but also the SPV maintainer code, wallet lifecycle maintainer code, and so on. That said, the `./keep-client relay` command is probably not a good fit here and we have basically two options to run that stuff:

Option 1: A bulk run of all maintainers with a single command. In this option, we would have a `./keep-client maintainer` command that will trigger all the maintainers in a single shot. This is more convenient but less flexible.

Option 2: A granular approach where each maintainer is triggered separately. In this option, we would have `./keep-client maintainer relay`, `./keep-client maintainer spv`, and so on. This is the exact opposite of the first option in terms of pros and cons.

I think we need to agree on a solution here @pdyraga @nkuba

Reply:
I like the idea of introducing the `maintainer` command. To decide between Option 1 and Option 2 we need to consider how much of the config and command's flags/args will be common for all the types of maintiners.

Maybe we could use something like:
`./keep-client maintainer --relay`
`./keep-client maintainer --spv`
`./keep-client maintainer --relay --spv`
`./keep-client maintainer` <- this one will run all the maintainers

Reply:
I like @nkuba's proposition.

Also, I think we should mark `maintainer` feature as experimental in help. This will be in-development feature and we'll release new versions in the meantime.

Reply:
Done. I changed the way the maintainer tasks are started.
Now it's started like that:
`KEEP_ETHEREUM_PASSWORD="password" ./keep-client maintainer  --relay  --config /path/to/config/maintainer.toml`

The `KEEP_ETHEREUM_PASSWORD` environment variable is needed because we use ethereum configuration to start the maintainer.

In the future more maintainers could be started by providing more flags:
`KEEP_ETHEREUM_PASSWORD="password" ./keep-client maintainer  --relay --spv  --config /path/to/config/maintainer.toml`

Reply:
Can we consider this resolved? I recall @tomaszslabon mentioned something about problems with the presented approach. What's the status here?

Reply:
@lukasz-zimnoch 
My concern was whether launching multiple maintainers with one command is really beneficial.
I left a TODO to consider whether or not we should do that.

But I don't think this issue is a blocker for this PR, as we have just one maintainer now.
If we decide we don't want multiple maintainers run within one process, we will have disallow providing keep-client with multiple maintainer flags.

Reply:
Ok, works for me! Thanks for clarification!